src/analytics/analytics-on-redshift.ts

```diff
@@ -304,6 +307,8 @@ export class RedshiftAnalyticsStack extends NestedStack {
       tablesOdsSource: props.tablesOdsSource,
       workflowBucketInfo: props.workflowBucketInfo,
       loadDataConfig: props.loadDataConfig,
+      odsSourceS3Bucket: props.dataSourceBucket,
```

Comment:
Is it really needed? We have bucket and prefix info in type `TablesODSSource`.

Reply:
change bucket and prefix to input parameter of step function workflows

Reply:
example:
  const tablesOdsSource: TablesODSSource = {
    event: {
      s3Bucket: props.dataSourceConfiguration.bucket,
      prefix: props.dataSourceConfiguration.prefix + TABLE_NAME_EVENT + '/',
      fileSuffix: props.dataSourceConfiguration.fileSuffix,
    },
    event_parameter: {
      s3Bucket: props.dataSourceConfiguration.bucket,
      prefix: props.dataSourceConfiguration.prefix + TABLE_NAME_EVENT_PARAMETER + '/',
      fileSuffix: props.dataSourceConfiguration.fileSuffix,
    },
    user: {
      s3Bucket: props.dataSourceConfiguration.bucket,
      prefix: props.dataSourceConfiguration.prefix + TABLE_NAME_USER + '/',
      fileSuffix: props.dataSourceConfiguration.fileSuffix,
    },
    item: {
      s3Bucket: props.dataSourceConfiguration.bucket,
      prefix: props.dataSourceConfiguration.prefix + TABLE_NAME_ITEM + '/',
      fileSuffix: props.dataSourceConfiguration.fileSuffix,
    },
  };